swift/Tasks/TasksListScreen.swift

```
@@ -55,6 +55,9 @@ class TasksListScreenViewModel: ObservableObject {
 
     private func startSync() throws {
         do {
+            Task {
+                try await dittoStore.execute(query: "ALTER SYSTEM SET DQL_STRICT_MODE = false")
+            }
```

Taken a closer look: I suspect the intention here is to bubble up any error happening within the Task scope to the caller of `startSync()`. Correct?

`Task` is performed asynchronously and errors are swallowed, i.e. go unnoticed. So if the query fails for some reason, sync will still be started and no error will be presented or anything.

If my assumption is correct, then you'd have to rewrite the code path and make it account for async execution and error handling.

---

I don't think that counts as an excuse tbh.